subprojects/docs/src/docs/release/notes.md

```diff
@@ -223,12 +268,6 @@ Gradle now places convention plugins under the `build-logic` directory instead o
 
 For more information about convention plugins, see [Convention Plugins](userguide/sharing_build_logic_between_subprojects.html#sec:convention_plugins).
```

Comment:
Piotr: Need to add why this is useful.

Reply:
@big-guy can you add?

Reply:
Short: Convention plugins is Gradle’s recommended way of organizing build logic.

Long: You can compose custom build logic by applying and configuring both core and external plugins and create custom convention plugins that define new project types and configure conventions specific to your project or organization.

Reply:
Thanks! added